service/src/test/java/bio/terra/drshub/controllers/VerifyPactDrsHubApiController.java

```diff
@@ -49,9 +52,11 @@
 @Tag("Pact")
 @WebMvcTest
 @ContextConfiguration(classes = {DrsHubApiController.class, PublicApiController.class})
-@Provider("drshub-provider")
+@Provider("drshub")
```

Comment:
Good catch here bringing the Pacticipant name in line with what's in Argo!

Do we need additional coordination with Cromwell to make this switch?  It is the only other Pacticipant in a relationship with `drshub`.

Do we need to re-register `drshub` into contract testing in Beehive?

Reply:
`Cromwell` is all set. The `drshub` has been enabled for contract testing in Beehive, no need to re-register.

Thanks for doing this review so quickly!